Dockerfile.openvino

```diff
@@ -26,13 +26,6 @@ RUN cmake -DCPACK_GENERATOR=DEB -DENABLE_PYTHON=ON -DENABLE_PYTHON_PACKAGING=ON
 RUN cmake --build /workspace/openvino_build --parallel 8
```

Comment:
above we clone OpenVINO of specific commit and repo.
Let's use OV commit from master once PA extension op is merged

Reply:
fixed